### ISSUE TITLE
feature request: Support for counting assigned IPs in the status field of the IPAddressPool object

### ISSUE THREAD

**Issue by cyclinder** [metallb/metallb#1638]
something like:
```
kind: IPAddressPool
metadata:
  name: default-pool
  namespace: kube-system
spec:
  addresses:
  - 10.6.168.140-10.6.168.150
  autoAssign: true
  avoidBuggyIPs: true
status:
  allocatedIPs:
    10.6.168.140:
       name: test-svc1
       namespace: default
    10.6.168.141:
       name: test-svc2
       namespace: default
     ...
```

@fedepaol any thoughts？ If it make sense, I'm interested in working on it :)

**Comment by fedepaol** [comment]
I have on my todo list to write a comprehensive proposal for exposing the status. I am gonna add this item and split the work in issues, but I'd prefer to have that out before starting to work on any item.

**Comment by cyclinder** [comment]
Thank for your reply.  Would you mind telling me when will do this？

**Comment by fedepaol** [comment]
It's out (and open for discussion https://github.com/metallb/metallb/pull/1685 )
Not sure I'd want to list all hte services under the status though.
Closing this in favour of the design proposal.
